assets/html/evolution/drop-nav-carousel.html

```diff
@@ -0,0 +1,503 @@
+    <link href="../../styles/main.css" rel="stylesheet">
+  </head>
+  <body>
+
+            <div class="evo_c-nav-drop-carousel">
+                <input type="radio" class="evo_c-nav-drop-carousel__radio" id="drop-1" name="drop-1"  />
+                <input type="radio" class="evo_c-nav-drop-carousel__radio" id="drop-2" name="drop-1"  />
+                <input type="radio" class="evo_c-nav-drop-carousel__radio" id="drop-3" name="drop-1"  />
+                <input type="radio" class="evo_c-nav-drop-carousel__radio" id="drop-4" name="drop-1"  />
+                <input type="radio" class="evo_c-nav-drop-carousel__radio" id="drop-5" name="drop-1" checked="checked" />
+                <!-- label_container -->
+                <div class="evo_c-nav-drop-carousel__label-wrap">
+                    <div class="evo_c-nav-drop-carousel__label-container">
+                        <label class="evo_c-nav-drop-carousel__drop-label" for="drop-1"></label>
+                    </div>
+                    <div class="evo_c-nav-drop-carousel__label-container">
+                        <label class="evo_c-nav-drop-carousel__drop-label" for="drop-2"></label>
+                    </div>
+                    <div class="evo_c-nav-drop-carousel__label-container">
+                        <label class="evo_c-nav-drop-carousel__drop-label" for="drop-3"></label>
+                    </div>
+                    <div class="evo_c-nav-drop-carousel__label-container">
+                        <label class="evo_c-nav-drop-carousel__drop-label" for="drop-4"></label>
+                    </div>
+                    <div class="evo_c-nav-drop-carousel__label-container">
+                        <label class="evo_c-nav-drop-carousel__drop-label" for="drop-5"></label>
+                    </div>
+                </div>
+                
+                <div class="evo_c-nav-drop-carousel__drop-content">
+                    
+                    <div class="evo_c-nav-drop-carousel__slide">
+                        <img src="images/boat.jpg" alt="boat" />
+                    </div>
+                    
+                    <div class="evo_c-nav-drop-carousel__slide">
+                        <img src="images/color-houses.jpg" alt="sky" />
+                    </div>
+                    
+                    <div class="evo_c-nav-drop-carousel__slide">
+                        <img src="images/other-world.jpg" alt="moutains" />
+                    </div>
+                    
+                    <div class="evo_c-nav-drop-carousel__slide">
+                        <img src="images/sand-castle.jpg" alt="castle" />
+                    </div>
+                    
+                    <div class="evo_c-nav-drop-carousel__slide">
+                        <img src="images/sun-set.jpg" alt="sun" />
+                    </div>
+                </div>
+                
+            </div>
+
+
+
+    <div class="evo_c-markup">
```

Comment:
@OscarRobertRodriguez 

Remove the complete `<div class="evo_c-markup">` section, we are not showing it anymore. Only the html for your component.